server/routes/api.js

```diff
@@ -12,6 +12,8 @@ router.get('/db/transaction', TransactionCtrl.getTransaction)
 router.post('/db/transaction', TransactionCtrl.createTransaction)
 
 router.get('/changelly/getCurrencies', ChangellyCtrl.getCurrencies)
+router.get('/changelly/getMinAmount/:from-:to', ChangellyCtrl.getMinAmount)
```

Comment:
is there any reason to have the "-"?

I would probably just write this: 
`/changelly/getMinAmount/:from/:to`